static/app/types/sampling.tsx

```diff
@@ -2,6 +2,7 @@ export enum DynamicSamplingBiasType {
   BOOST_ENVIRONMENTS = 'boostEnvironments',
   BOOST_LATEST_RELEASES = 'boostLatestRelease',
   BOOST_KEY_TRANSACTIONS = 'boostKeyTransactions',
+  BOOST_LOW_VOLUME_TRANSACTIONS = 'prioritiseByTxName',
```

Comment:
```suggestion
  BOOST_LOW_VOLUME_TRANSACTIONS = 'boostLowVolumeTransactions',
```
i'll change in my PR too.